readthedocs/donate/views.py

```diff
@@ -54,3 +56,15 @@ def get_queryset(self):
 
     def get_template_names(self):
         return [self.template_name]
+
+
+def click_proxy(request, promo_id):
```

Comment:
I don't think we'll be able to do CSRF on the image loading. That will be a request sent by the browser, though I guess we could validate on a cookie, but we don't want sessions for logged out users..